docs/usage.md

```diff
@@ -111,6 +111,57 @@ chr21:16609476_A_G 16609476 A G
 chr21:16609525_T_A 16609525 T A
 ```
 
+## Samplesheet genetic map
+
+To improve imputation quality, you can optionally provide a genetic map. This allows the software to better estimate recombination rates across each chromosome. Use the `--map` parameter to specify the location of the genetic map file. The file must be a comma-separated table with at least two columns and a header row, as shown below:
+
+```bash
+--map '[path to samplesheet file]'
+```
+
+### Structure
+
+A typical genetic map samplesheet contains one row per chromosome.
+For example, a file specifying maps for two chromosomes might look like this:
+
+```console title="map.csv"
+chr,map
+chr21,GRCh38_chr21.glimpse.map
+chr22,GRCh38_chr22.glimpse.map
+```
+
+Genetic map files are available in various formats, so an autoconversion module has been implemented. A valid map file for `nf-core/phaseimpute` must:
+
+- be a text file (compressed or uncompressed),
+- contain at least the following two columns:
+  - `pos`: physical position (base pairs)
+  - `cm`: genetic position (centiMorgans)
+- additional optional columns include:
+  - `chr`: chromosome name
+  - `id`: variant identifier
+
+You can configure how your map file is interpreted using the following parameters:
+
+- `--map_sep`: Field separator used in the map file (e.g. "\t", " ", ",")
+- `--map_header`: Whether the file contains a header row (`true` or `false`)
+- `--map_col_names`: Ordered list of column names in the file
```

Comment:
I think these could be easily inferred in the module so that we don't add new params to users (which means they might be more prone to errors).